src/engine/server/server.cpp

```diff
@@ -1042,11 +1049,11 @@ int CServer::DelClientCallback(int ClientID, int Type, const char *pReason, void
 	
 	//Keep information about client for 10 minutes
 	pThis->m_NetSession.AddSession(pThis->m_NetServer.ClientAddr(ClientID), 10*60, &pThis->m_aClients[ClientID].m_Session);
-	dbg_msg("infclass", "session created for the client %d", ClientID);
+	//dbg_msg("infclass", "session created for the client %d", ClientID);
```

Comment:
why was this and below commented out?

Reply:
I think this dbg_msg was added to debug code and someone forgot to remove it again. But maybe i am wrong and the message is useful?

Reply:
Oh btw do you know does this work correctly? If someone gets accused via ban player, but the vote doesnt start and he then leaves and comes back within 30 minutes, will he still have this one accusation from before?